libs/model/src/user/GetNewContent.query.ts

```diff
@@ -0,0 +1,74 @@
+import { Query } from '@hicommonwealth/core';
+import * as schemas from '@hicommonwealth/schemas';
+import { QueryTypes } from 'sequelize';
+import { sequelize } from '../database';
+
+export function GetNewContent(): Query<typeof schemas.GetNewContent> {
+  return {
+    ...schemas.GetNewContent,
+    auth: [],
+    body: async ({ actor }) => {
+      const joinedCommunityIdsWithNewContent: string[] = [];
+
+      // get ids of communities where user has membership and that community got new threads/comments
+      // since user's last active session per address (the address that user used to join that community)
+      const response = await sequelize.query<{ community_id: string }>(
+        `
+          WITH CommunityContent AS (
```

Comment:
You can probably optimize this query a little bit more, basically removing the subqueries and CTE... something like: (just make sure this is equivalent pls)

```
 
  SELECT 
    a.community_id
  FROM 
    "Addresses" a
    LEFT JOIN "Threads" t  ON t.community_id = a.community_id 
      AND t.created_at > a.last_active 
      AND t.deleted_at IS NULL
    LEFT JOIN "Comments" c  ON c.community_id = a.community_id 
      AND c.created_at > a.last_active 
      AND c.deleted_at IS NULL
  WHERE 
    a.verified IS NOT NULL 
    AND a.last_active IS NOT NULL 
    AND a.user_id = 131317
  GROUP BY 
    a.community_id
 HAVING  COUNT(t.id) + COUNT(c.id) > 0;
```